js-exercises/rot-13/rot13.js

```diff
@@ -0,0 +1,36 @@
+const rot13 = (cipheredText) => {
+  if (!isString(cipheredText)) {
+    throw new Error(
+      "Invalid Input Received. Exprected String. But, received " +
+        typeof cipheredText
+    );
+  }
+  let decipheredText = "";
+  for (const character of cipheredText) {
+    if (isCapitalAlphabet(character)) {
+      decipheredText = decipheredText + getRotatedUpperCase(character);
+    } else {
+      decipheredText = decipheredText + character;
+    }
+  }
+  return decipheredText;
+};
+
+const isString = (input) => {
+  return typeof input === "string";
+};
+
+const isCapitalAlphabet = (character) => {
+  return !(character == character.toLowerCase());
```

Comment:
Why did you chose to use the loose equality here?

Reply:
I thought it's sufficient. Strict equality makes more sense. I've updated.